README.md

```diff
@@ -1,3 +1,33 @@
-# dissent-app
-The dissent app enables organizations to restrict actions to only those who have not expressed approval in recent votes.
+# Dissent-Voting <img align="right" src="https://github.com/1Hive/website/blob/master/website/static/img/bee.png" height="80px" />
+
+The dissent-voting app enables organizations to restrict voting to those who have not expressed approval in recent votes. This is achieved through a combination of the [Dandelion Voting](https://github.com/1Hive/dissent-voting-app/tree/master/dandelion-voting) and [Dissent Oracle](https://github.com/1Hive/dissent-voting-app/tree/master/dissent-oracle). Dandelion Voting keeps track of who has voted `yay` or `nay` on each proposal and when that vote took place. The Dissent Oracle then checks whether an account has voted `yay` on a proposal within the time-frame of the dissent-window and returns a boolean expressing that fact. This allows other applications (such as [Redemptions](https://github.com/1Hive/redemptions-app/)) to restrict actions based on the response from the Dissent Oracle.
```

Comment:
> or `nay`

We only keep track of `yay` votes afaik.

Reply:
Good call! Thanks